src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -54,7 +54,7 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
 
   const toggleIsScheduleOpen = () => {
     setIsScheduleOpen((wasOpen) => {
-      localStorage.setItem(scheduleOpenKey, String(!isScheduleOpen));
+      localStorage.setItem(scheduleOpenKey, String(isScheduleOpen));
```

Comment:
Looks like you're storing the string `'profile.schedule.isOpenOrNot'`, instead of the `true`/`false` that is intended. Not sure how I missed that on the previous PR, sorry.
```suggestion
      localStorage.setItem(scheduleOpenKey, String(!wasOpen));
```

Fixing this here should make the change to the default value of the `useState` above unnecessary.